src/i18next/init-i18next.js

```diff
@@ -0,0 +1,55 @@
+import i18n from "i18next";
+import i18nextXhrBackend from "i18next-xhr-backend";
+import { initReactI18next } from "react-i18next";
+import ICU from "i18next-icu";
+
+(i18n.default || i18n)
+  .use(i18nextXhrBackend)
```

Comment:
I've set up i18next to download translation files lazily, as needed, on a per module basis, via i18next-xhr-backend.